versioned_docs/version-2.0.0/keploy-explained/docs-dev-guide.md

```diff
@@ -0,0 +1,127 @@
+
+Now you can build and view the site locally:
+
+```bash
+yarn start
+```
+
+The command starts a local development server and opens a browser window.
+
+## Prettier
+
+1. Fork the repository
+
+2. Clone the repository with the following command. Replace the <GITHUB_USERNAME> with your username
+
+```sh
+git clone https://github.com/<GITHUB_USERNAME>/docs.git
+```
+
+3. Go into the directory containing the project
+
+```sh
+cd docs
+```
+
+4. Install all the dependencies
+
+```sh
+yarn
+```
+
+5. Start the development server.
+
+```sh
+yarn start
+```
+
+6. To reformat:
+
+```
+yarn prettier --write '**/*.{js,md}'
+```
+
+When we merge your PR, a new build automatically occurs and your changes publish to [https://docs.keploy.io](https://docs.keploy.io).
+
+## How to Contribute Code?
+
+Bug fixes, performance improvements, code formatting, ...
+There are a lot ways in which you can contribute code!
+The issues list of a project is a great place to find something that you can help us with.
+
+To increase the chances of your code getting merged, please ensure that:
+
+- You satisfy our contribution criteria
+- Your pull request:
+  - Passes all checks and has no conflicts.
+  - Has a well-written title and message that briefly explains your proposed changes.
+
+## How to Report Bugs, Provide Feedback or Request Features?
+
+We welcome all kinds of bug reports, user feedback and feature requests! We've created some issue templates to assist you in this. Please use them to create a new issue in the relevant project's repository. If you are not sure how to create an issue, here are the steps:
+
+1. Navigate to the main page of the documentation website repository
+
+2. Click `Issues`
+
+3. Click `New issue`
+
+4. Our repository uses issue templates, click `Get started` next to the type of issue you'd like to open
+
+5. Type a title for your issue. We advise you to stick to the issue template to describe the issue
+
+6. After you are finished, click `Submit new issue`
+
+## How to Open a Pull Request to Add New Documentation or Fix Bugs?
+
+After you have cloned the documentation repo to your computer locally, edit the files that you wish to change and follow the steps to open a pull request.
+
+1. `add` and `commit` your changes to your repository
+
+2. Git push your changes to your fork on Github
+
+3. On Github, click on the `Create a new Pull Request` button.
+
+4. Use a succinct title and descriptive comments to describe your Pull Request. Also Ensure maintainers can understand your proposed changes from the description
+
+## How to Suggest UI/UX Improvements?
+
+One of the most important areas of improvement to our software Keploy UI. We really need your help with this!
+
+If you have ideas on how we can improve, please share them with us by creating a [new issue](https://github.com/keploy/ui/issues/new/choose).
```

Comment:
I see a broken link.

Reply:
yes 
Copied this from here https://github.com/keploy/docs/blob/419f33a78ae0bc29362d7b58fe9cc54e845e0bf6/CONTRIBUTING.md?plain=1#L121

So we can create a issue for this as it is not working in the main code also

Reply:
need to remove `/ui` link only from here

Reply:
[#924](https://github.com/keploy/keploy/issues/924)